.github/workflows/smoke-test.yml

```diff
@@ -11,13 +11,25 @@ jobs:
     name: smoke-tests-${{ github.event.inputs.environment }}
     runs-on: ubuntu-latest
     steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+
+      - name: install bundle
+        run: |
+          gem install bundler
```

Comment:
so do we want to install rspec separately here or do a bundle with a different Gemfile?